src/java/picard/cmdline/CommandLineParser.java

```diff
@@ -398,6 +418,9 @@ public void htmlPrintOptions(final PrintStream stream, final boolean printCommon
      */
     private void htmlPrintOptionTableRows(final PrintStream stream, final boolean printCommon) {
         for (final OptionDefinition optionDefinition : optionDefinitions) {
+
+            checkForNonASCII(optionDefinition.doc, optionDefinition.name);
```

Comment:
what about in the command line tool's description?  Does this cover that case?  It would be nice to have any printed text be checked for non-ASCII characters and html codes.